test/distributed/test_c10d_gloo.py

```diff
@@ -152,7 +152,7 @@ def simple_coalesced_reduce_tests(rank, world_size):
         (
             c10d.ReduceOp.MAX,
             [torch.tensor([rank + x]) for x in [1.0, 2.0]],
-            [torch.tensor([world_size]), torch.tensor([world_size + 1.0])],
+            [torch.tensor([world_size + 0.0]), torch.tensor([world_size + 1.0])],
```

Comment:
nit: any difference between `world_size + 0.0` here and the `float(world_size)` on line 78? 

Wondering if one should be changed to the other for consistency

Reply:
Updated.